src/org/thoughtcrime/securesms/WelcomeActivity.java

```diff
@@ -121,7 +121,7 @@ private void startImportBackup() {
                 .withPermanentDenialDialog(getString(R.string.perm_explain_access_to_storage_denied))
                 .onAllGranted(() -> {
                     File imexDir = DcHelper.getImexDir();
-                    if (Build.VERSION.SDK_INT >= 29) {
+                    if (Build.VERSION.SDK_INT >= 30) {
```

Comment:
Note that this line changes importing backups, while the bug in the testing group was with exporting.

But if things work, that's fine.

And, yes, I also remember finding the documentation unclear.

Reply:
> Note that this line changes importing backups, while the bug in the testing group was with exporting.

yip, but the issue with andorid10 was also that the import does not find the file just exported ... the selector was just empty, even if export run through.

Reply:
I'm afraid this is an issue with both Android 10 and 11 :/

Probably the issue is that when you try to select a file, it puts you into the "Recent Files" screen, which is empty. 

But then if you go to the hamburger menu and then select "Downloads", you will find the exported file. It will be neatly at the top of the list as the list is sorted by "Last Modification (recent first)".

What we _could_ do is say the system that it shall show the "Downloads" folder first. But then the drawback is that the backup will be somewhere down in the list as the list is sorted by "Name (A-Z)".

Maybe the unfortunate sort order would be the lesser evil though, idk.

Reply:
> I'm afraid this is an issue with both Android 10 and 11 :/

on android 11 i could import the exported backup, this did not work on android 10, at least not directly. might be that it was hidden somehow, yes, tbh, i did not searched too much for it (as it appears instantly on android11 i did not even had that idea :)